pom.xml

```diff
@@ -54,6 +54,8 @@
         <version.modelmapper>3.2.0</version.modelmapper>
```

Comment:
Why modelmapper dependency does not removed?

Reply:
I'll remove it today or latest tomorrow morning, since there are also a lot of tests to be adjusted.